docroot/themes/humsci/humsci_basic/patterns/date-stacked-vertical-card/date-stacked-vertical-card.html.twig

```diff
@@ -58,11 +58,7 @@
     {% endif %}
 
     {% if url %}
-      <a href="{{ url }}" class="hb-card__decorative-link">
-        <svg aria-label="external link" xmlns="http://www.w3.org/2000/svg" width="23" height="24" viewBox="0 0 23 24">
-          <path d="M11 23l-2-2 7-7H0v-3h16L9 3l2-2 10 10 1 1-11 11z"/>
-        </svg>
-      </a>
+      <a href="{{ url }}" class="hb-card__decorative-link" aria-label="link to content"></a>
```

Comment:
for this aria-label, you should do some string interpolation:

`aria-label="#{url}"` at least that will provide a user more context than hearing 'link to content 10 times :)

Reply:
Good idea! This links is currently the same as the link in the other patterns (horizontal card, vertical card, etc). I'm thinking this fix should be applied to the others as well.